tests/test_writer.py

```diff
@@ -10,7 +10,7 @@
 from arc.common import read_yaml_file
 
 from t3.common import DATA_BASE_PATH, EXAMPLES_BASE_PATH
-from t3.schema import InputBase, RMG
+from t3.schema import InputBase, RMG, T3
```

Comment:
Commits that modify tests should start with `Tests: `, see the T3 [commit style](https://github.com/ReactionMechanismGenerator/T3/wiki/Contributor-Guidelines#commits-style).